json-java8/shared/src/main/scala/rapture/json/java8/time.scala

```diff
@@ -0,0 +1,81 @@
+package rapture.json.java8
+
+import java.time._
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeFormatter._
+
+import rapture.core.{StringParser, StringSerializer}
+import rapture.json.JsonAst
+
+private[java8] trait TimeSerializers {
+
+  implicit final def localDateSerializer(implicit ast: JsonAst): StringSerializer[LocalDate] = serializeLocalDate(ISO_LOCAL_DATE)
+  implicit final def localDateTimeSerializer(implicit ast: JsonAst): StringSerializer[LocalDateTime] = serializeLocalDateTime(ISO_LOCAL_DATE_TIME)
+  implicit final def localTimeSerializer(implicit ast: JsonAst): StringSerializer[LocalTime] = serializeLocalTime(ISO_LOCAL_TIME)
+  implicit final def zonedDateTimeSerializer(implicit ast: JsonAst): StringSerializer[ZonedDateTime] = serializeZonedDateTime(ISO_ZONED_DATE_TIME)
+  implicit final def offsetDateTimeSerializer(implicit ast: JsonAst): StringSerializer[OffsetDateTime] = serializeOffsetDateTime(ISO_OFFSET_DATE_TIME)
+  implicit final def offsetTimeSerializer(implicit ast: JsonAst): StringSerializer[OffsetTime] = serializeOffsetTime(ISO_OFFSET_TIME)
+
+  final def serializeLocalDate(formatter: DateTimeFormatter)(implicit ast: JsonAst): StringSerializer[LocalDate] = {
```

Comment:
Do we actually need access to the `JsonAst` implicit for these? Could it be removed?